src/find_ice_labels.jl

```diff
@@ -1,4 +1,30 @@
 using ImageBinarization: AbstractImageBinarizationAlgorithm, binarize
+using Images: build_histogram
+using Peaks: findmaxima, peakproms!, peakwidths!
+
+"""
+Given the edges and counts from build_histogram, identify local maxima and return the location of the
+largest local maximum that is bright enough that it is possibly sea ice. Locations are determined by 
+the edges, which by default are the left bin edges. Note also that peaks defaults to the left side of
+plateaus. Returns Inf if there are no non-zero parts of the histogram with bins larger than the possible
+ice threshold, or if there are no detected peaks larger than the minimum prominence.
+"""
+function get_ice_peaks(edges, counts; possible_ice_threshold::Float64=0.30, minimum_prominence::Float64=0.05, window::Int64=3)
+    size(counts)
+    counts = counts[1:end]
+    normalizer = sum(counts[edges .> possible_ice_threshold])
+    # Normalize the possible sea ice section of the histogram. 
+    # Images with a lot of masked pixels can have large peaks at 0, which
+    # we don't want to include in the normalization. If no potential
+    # ice pixels, then return early
+    counts = normalizer > 0 ? counts ./ normalizer : return Inf
+    pks = findmaxima(counts, window) |> peakproms! |> peakwidths!
+    pks_df = DataFrame(pks[Not(:data)])
+    pks_df = sort(pks_df, :proms, rev=true)
+    maximum(pks_df.proms) < minimum_prominence && return Inf
+    return edges[pks_df[argmax(pks_df.proms), :indices]] 
```

Comment:
You could use `findmax` here which produces both the max and argmax.
```suggestion
    mx, argmx = findmax(pks_df.proms)
    mx < minimum_prominence && return Inf
    return edges[pks_df[argmx, :indices]] 
```

Reply:
very cool, good to know!